verhoeven2021replication/journal/metadata.yaml

```diff
@@ -5,59 +5,51 @@
 #  - For a successful replication, it should be prefixed with "[Re]"
 #  - For a failed replication, it should be prefixed with "[¬Re]"
 #  - For other article types, no instruction (but please, not too long)
-title: "ReScience (R)evolution"
+title: "[Re] Replication Study of 'Generative causal explanations of black-box classifiers'"
 
 # List of authors with name, orcid number, email and affiliation
 # Affiliation "*" means contact author (required even for single-authored papers)
 authors:
-  - name: Konrad Hinsen
-    orcid: 0000-0003-0330-9428
-    email: konrad.hinsen@cnrs.fr
-    affiliations: 1,2
-    
-  - name: Nicolas P. Rougier
-    orcid: 0000-0002-6972-589X
-    email: Nicolas.Rougier@inria.fr
-    affiliations: 3,4,5,*      # * is for contact author
+  - name: Ivo Verhoeven
+    orcid: 0000-0002-5163-210X
+    email: ivo.verhoeven@student.uva.nl
+    affiliations: 1, *
+authors:
```

Comment:
This yaml is malformed. `authors` is a list, which contains objects. You have created multiple `authors` lists.. please remove this line (as `authors` list is defined in L12)